src/command_line_parser.cc

```diff
@@ -1815,8 +1889,7 @@ CLParser::VerifyOptions()
   if (params_->model_name.empty()) {
     Usage("Failed to parse -m (model name). The value must be specified.");
   }
-  if (params_->concurrency_range.start <= 0 ||
-      params_->concurrent_request_count < 0) {
```

Comment:
`concurrent_request_count` was part of the deprecation options which were cleaned up and now instead use the `concurrency_range` data structure to hold state.